docs/linking_to_external_code/import_maps.md

```diff
@@ -42,7 +42,7 @@ $ deno run --import-map=import_map.json --unstable color.ts
 
 To use starting directory for absolute imports:
 
-```json
+```json5
 // import_map.json
```

Comment:
There is no good reason that there is a comment in this block, and should be moved outside instead.